examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -165,26 +178,27 @@ def evaluate(model, embed_layer, eval_loader, node_feats):
     embed_layer.eval()
     eval_logits = []
     eval_seeds = []
- 
+
     with th.no_grad():
+        th.cuda.empty_cache()
         for sample_data in tqdm.tqdm(eval_loader):
-            th.cuda.empty_cache()
             seeds, blocks = sample_data
+
             feats = embed_layer(blocks[0].srcdata[dgl.NID],
-                    blocks[0].srcdata[dgl.NTYPE],
-                    blocks[0].srcdata['type_id'],
-                    node_feats)
+                                blocks[0].srcdata['ntype'],
```

Comment:
why do you replace `dgl.NTYPE` with "ntype"?

Reply:
Yes.